install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -234,6 +224,40 @@ spec:
               fi
 {{- end }}
 
+              # AKS: If azure-vnet is installed on the node, and (still) configured in bridge mode,
+              # configure it as 'transparent' to be consistent with Cilium's CNI chaining config.
+              # If the azure-vnet CNI config is not removed, kubelet will execute CNI CHECK commands
+              # against it every 5 seconds and write 'bridge' to its state file, causing inconsistent
+              # behaviour when Pods are removed.
+              if stat /etc/cni/net.d/10-azure.conflist > /dev/null 2>&1; then
+
+                echo "azure-vnet configured in bridge mode. Changing to 'transparent'.."
+                sed -i 's/"mode":\s*"bridge"/"mode":"transparent"/g' /etc/cni/net.d/10-azure.conflist
```

Comment:
I observe that `mode` in the pattern to match starts with a lowercase letter, when the one you removed in the previous commit had a `M`. I note you're not searching the same file, but this is just to be sure that this is intended.

Reply:
Yup, different file, different format.